aspnetcore/fundamentals/routing.md

```diff
@@ -203,6 +210,8 @@ A few differences exist between endpoint routing and versions of routing earlier
   | `/search/{*page}`  | `/search/admin%2Fproducts` (the forward slash is encoded)             |
   | `/search/{**page}` | `/search/admin/products`                                              |
 
+* The endpoint routing system is lower level than the MVC routing system.
```

Comment:
Hmmm, I don't know about this. In 3.0, MVC routing is endpoint routing. On app startup MVC provides a data source of all its configured routes to endpoint routing.